lookout/style/format/rule_stat.py

```diff
@@ -21,13 +25,15 @@ def __init__(self):
         self.pred_classes = [0 for _ in range(len(CLASSES))]
 
 
-def print_rule_table(rule_stat):
+def generate_rule_table(rule_stat: Mapping[Any, RuleStat]) -> AnyStr:
```

Comment:
`AnyStr = TypeVar('AnyStr', str, bytes)`

We don't work with bytes! Please replace all those to `str`